operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -347,7 +118,7 @@ See [Troubleshooting](Track_the_Status_of_a_Session.md) for more information.
     10.252.1.11
     ```
 
-1. Run readiness checks.
+## Run readiness checks.
 
     After worker node personalization, in order to verify the overall readiness of the iSCSI targets before booting compute nodes or UANs,
```

Comment:
This final check says that it should be done after personalization to verify the readiness. But no information is given about when or why the previous checks would be run. The introductory text at the top of the page should provide guidance on which of these checks should be considered routine health checks vs those that are something else (troubleshooting steps? More in-depth checks? Manual steps that are also covered by automated Goss tests?)

Reply:
Marking this resolved because the work on the verification page is moving to [CASMPET-7640](https://jira-pro.it.hpe.com:8443/browse/CASMPET-7640). Once a PR is open for that, I'll add comments for the relevant issues identified in this PR.